AGENTS.md

```
@@ -81,39 +152,445 @@ npm run local -- eval -c path/to/config.yaml
+   gh pr create --title "Your PR Title" --body "PR description"
+   ```
+
+7. **Wait for review and CI checks** before merging
+
+## Pull Request Titles
+
+Pull request titles must follow Conventional Commits format. PR titles become squash-merge commit messages and changelog entries, so consistency is critical.
+
+### Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)